Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2_test.py

```diff
@@ -0,0 +1,203 @@
+import json
```

Comment:
More of a general UT comment.
I always prefer creating UTs for helper functions with more pre-defined behaviours than the actual commands, which might be better suited for testing via TPB anyway.
IMO that's what makes UTs really powerful - testing Pythonic logic rather than mocking HTTP requests.
Leaving this to the judgement of @bakatzir and you